cax/tasks/filesystem.py

```diff
@@ -17,41 +17,70 @@ class SetPermission(Task):
     """Set the correct permissions at the PDC in Stockholm"""
 
     def __init__(self):
-        self.chmod_file = '/cfs/klemming/projects/xenon/misc/basic_file'
-        self.chmod_folder = '/cfs/klemming/projects/xenon/misc/basic'
+        
+        self.raw_data = {"tegner-login-1": "/cfs/klemming/projects/xenon/xenon1t/raw/",
+                    "midway-login1": "/project/lgrandi/xenon1t/raw/"}
+        
+        self.proc_data = {"tegner-login-1": "/cfs/klemming/projects/xenon/xenon1t/processed/",
+                    "midway-login1": "/project/lgrandi/xenon1t/processed/"}
```

Comment:
Looking at this more carefully now: shouldn't you try to grab these from the cax.json to minimize hardcoding?